pages/[lng]/blog/[slug].tsx

```diff
@@ -138,14 +145,18 @@ const BlogSlug: FC<any> = ({
 
 export const getServerSideProps: GetServerSideProps = async ({
   params,
-  req,
+  res,
+  req
 }) => {
-  const brand = await useBrand(req);
+  const [brand, headerImage] = await Promise.all([
+    useBrand(req),
+    getBlogHeaderImage(GRAPHQL_URI(req)),
+    useAuthToken({ req, res, env: process.env })
+  ])
   const { slug } = params;
   const urlSite = `https://${req.headers.host}/${params.lng}/blog/${slug}`;
   const defaultLanguage = brand?.settings?.defaultLanguage || params.lng || 'id';
   const { default: lngDict = {} } = await import(`locales/${defaultLanguage}.json`);
```

Comment:
```suggestion
  const { slug } = params
  const urlSite = `https://${req.headers.host}/${params.lng}/blog/${slug}`
  const defaultLanguage = brand?.settings?.defaultLanguage || params.lng || 'id'
  const { default: lngDict = {} } = await import(`locales/${defaultLanguage}.json`)
```

biar seragam jangan pake titik koma pacc